build_all.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
```

Comment:
This needs a lot more context. Who are you intending to run it? What are the dependencies? What it is it supposed to do when somebody runs it?